src/Utils.ts

```diff
@@ -23,17 +25,22 @@ const Utils = {
    * This method submits a raw transaction and returns transaction hash.
    * @param tx Raw transaction.
    * @param txOption Transaction options.
+   * @param web3 The web3 instance to be used for fetching nonce.
    */
-  async sendTransaction(tx: any, txOption: any): Promise<string> {
+  async sendTransaction(tx: any, txOption: any, web3: Web3): Promise<string> {
```

Comment:
Should we also add test for sendTransaction method?
I know it should have been written earlier 😞

Reply:
We should :) but would request to add it outside of scope of this PR.